interfaces/dbus/backend.go

```diff
@@ -55,21 +57,36 @@ func (b *Backend) Name() interfaces.SecuritySystem {
 	return "dbus"
 }
 
+func shouldCopyConfigFiles(snapInfo *snap.Info) bool {
+	// Only copy config files on classic distros
+	if !release.OnClassic {
+		return false
+	}
+	// Only copy config files if we have been reexecuted
+	if reexecd, _ := snapdtool.IsReexecd(); !reexecd {
+		return false
+	}
+	switch snapInfo.Type() {
+	case snap.TypeOS:
+		// fugly - but we need to make sure that the content of the
+		// "snapd" snap wins
+		//
+		// TODO: this is also racy but the content of the files in core and
+		// snapd is identical cleanup after link-snap and
```

Comment:
I cannot parse `.. is identical cleanup after ...`  - could you please rephrase that.

In addition. Can you please remove `fugly`, perhaps replace that with `XXX: `

Reply:
Note that this is an existing comment that I just moved as part of the PR.

The comment seems to make sense if I add a full stop after "is identical", so that's what I've done.